packages/protocol/contracts/bridge/libs/LibBridgeRetry.sol

```diff
@@ -61,6 +61,7 @@ library LibBridgeRetry {
 
         // successful invocation
         if (
+            // TODO(dave): do we want to use gasleft() here? couldn't it consume beyond the gas limit?
```

Comment:
The message.gasLimit only apply for `processMessage`, once it fails, whoever calls the `retryMessage` will use the tx's gasLimit, the message.gasLimit no longer apply.